### DISCUSSION TITLE
Media file with several audio tracks

### DISCUSSION THREAD

**Discussion by MetaHG** [absadiki/subsai#24] [Q&A]
Hi,

I am trying to run this tool on media files that can have several audio tracks. Typically, each track corresponds to a different language that can be selected by the user (in VLC for example). When running the tool, I noticed that only the first track is selected by default. Is there a way to specify which track to select, both from the CLI and UI?

**Comment by absadiki** [comment]
Hi @MetaHG,

The tool does not support media files with more than one audio track currently. 

But you can easily split them manually with `ffmpeg` using [-map](http://ffmpeg.org/ffmpeg.html#Advanced-options), you can them run the CLI batch transcription for all audio tracks.
